### ISSUE TITLE
Removing a disabled package causes error

### ISSUE THREAD

**Issue by pritianka** [wbond/package_control#822]
I'm getting this exception in the console when removing a package:

```
theme loaded
reloading Packages/User/Package Control.sublime-settings
Traceback (most recent call last):
  File "package_control.commands.remove_package_command in /Users/priyankasharma/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 76, in unignore_package
  File "package_control.package_disabler in /Users/priyankasharma/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 145, in reenable_package
  File "package_control.events in /Users/priyankasharma/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 69, in clear
  File "package_control.events in /Users/priyankasharma/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 64, in do_clear
KeyError: 'WakaTime'
```

Also, that package isn't installing correctly which could be related.

**Comment by pritianka** [comment]
Found from #834, the package was disabled somehow.

**Comment by wbond** [comment]
Fixed by 3ea1edff5be0c95d4f621fc79abd39aa0ba4e964
